yt_dlp/extractor/dumpert.py

```diff
@@ -10,7 +10,7 @@ class DumpertIE(InfoExtractor):
     _VALID_URL = r'''(?x)
         (?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl(?:
             /(?:mediabase|embed|item)/|
```

Comment:
we should probably move the slash that trails the TLD out of the optional group so we don't accidentally match the wrong domain e.g. `dumpert.nlmaogetpwned.xyz?selectedId=123_abc`
```suggestion
        (?P<protocol>https?)://(?:(?:www|legacy)\.)?dumpert\.nl/(?:
            (?:mediabase|embed|item)/|
```